yarn.lock

```diff
@@ -1725,10 +1725,6 @@ cipher-base@^1.0.0, cipher-base@^1.0.1, cipher-base@^1.0.3:
   dependencies:
     inherits "^2.0.1"
 
-circular-dependency-plugin@^3.0.0:
```

Comment:
@hotzenklotz Should this commit remove the circular-dependency-plugin completely or only from yarn.lock? I'm still getting circular dependency errors in webpack. I don't think removing them is crucial, but jenkins complains about them. What is the plan for that?